socorro/unittest/lib/test_threaded_task_manager.py

```diff
@@ -132,15 +124,9 @@ def insert_into_list(anItem):
         try:
             ttm.start()
             time.sleep(0.2)
-            ok_(len(ttm.thread_list) == 2,
-                "expected 2 threads, but found %d"
-                % len(ttm.thread_list))
-            ok_(len(my_list) == 5,
-                'expected to do 5 inserts, '
-                'but %d were done instead' % len(my_list))
-            ok_(sorted(my_list) == range(5),
-                'expected %s, but got %s' % (range(5),
-                                             sorted(my_list)))
+            assert len(ttm.thread_list) == 2
+            assert len(my_list) == 5
+            assert sorted(my_list) == range(5)
```

Comment:
+1

Reply:
removing the text is fine. With or without it I have to go lookup this part of the test suite when things fail.

Reply:
Oh--but that's the magic of pytest. pytest will show you the assertion error, then walk the stack and show you the line, the values, where the values came from, and some other information. But it only shows you that information if there's no assertion message.

Check out the top of https://docs.pytest.org/en/latest/ for a basic example.

`eq_` and `ok_`, would generate the assertion message for you, but they didn't walk the stack and show you as much information as pytest does.